test-data/unit/check-modules.test

```diff
@@ -847,3 +849,17 @@ import x
 class Sub(x.Base):
     attr = x.Base.attr
 [out]
+
+[case testImportCycleStability3]
```

Comment:
Another potential test case is one where two files that form a cycle symmetrically refer to a attribute defined in the other file. This would remain a valid test case no matter in which order we'll end up type checking the modules.